src/components/pickers/TimePicker.js

```diff
@@ -171,16 +171,18 @@ export default {
       } else if (allowed instanceof Function) {
         return allowed(value)
       } else if (allowed instanceof Object) {
-        const range = type === 'minute' && this.ranges.minutes || !this.is24hr && this.ranges.hours.slice(1, 13) || this.ranges.hours
+        const range = type === 'minute' && this.ranges.minutes ||
+          !this.is24hr && this.ranges.hours.slice(1, 13) ||
```

Comment:
It limits the range of allowable hours to 1-12 when not in 24hr mode. Although now that I think about it that gives me no way of distinguishing between AM/PM hours. 12hr format is the worst.